test/Libraries/Microsoft.Extensions.AI.Abstractions.Tests/Contents/AIContentTests.cs

```diff
@@ -31,5 +32,25 @@ public void Constructor_PropsRoundtrip()
         Assert.Same(props, c.AdditionalProperties);
     }
 
-    private sealed class DerivedAIContent : AIContent;
+    [Fact]
+    public void Serialization_Roundtrips()
+    {
+        AIContent original = new()
+        {
+            RawRepresentation = new object(),
+            AdditionalProperties = new AdditionalPropertiesDictionary { { "key", "value" } }
+        };
+
+        Assert.NotNull(original.RawRepresentation);
+        Assert.Single(original.AdditionalProperties);
+
+        string json = JsonSerializer.Serialize(original, AIJsonUtilities.DefaultOptions.GetTypeInfo(typeof(AIContent)));
```

Comment:
Have we enabled the AOT analyzer in test projects? I'd be surprised if we couldn't pass the JSO directly here.

Reply:
I didn't check. I'm just in the habit of this now in these projects.